internal/pkg/plugin/trello/trello.go

```diff
@@ -0,0 +1,62 @@
+package trello
+
+import (
+	"fmt"
+
+	"github.com/merico-dev/stream/pkg/util/log"
+	"github.com/merico-dev/stream/pkg/util/trello"
+)
+
+// Options is the struct for configurations of the trellogithub plugin.
+type Options struct {
+	Owner           string
+	Repo            string
+	KanbanBoardName string
+}
+
+type TrelloItemId struct {
+	boardId     string
+	todoListId  string
+	doingListId string
+	doneListId  string
+}
+
+// CreateTrelloItems create board/lists, and set secret by ids
+func CreateTrelloItems(options *Options) (*TrelloItemId, error) {
```

Comment:
As far as I can see from the code, this piece of code only creates a "board"; no "lists" are created. So, the comment might not be very accurate. Also, the name of the function is "CreateTrelloItems". What does "Item" mean here? Does "Item" mean both "board" and "lists"? Is this an official usage? I personally suggest a better function name.